MAUI/ImageEditor/getting-started.md

```diff
@@ -307,4 +307,98 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
 {% endhighlight %}
 {% endtabs %}
 
-N> The size value will only be available after the image has been loaded into view.
\ No newline at end of file
+N> The size value will only be available after the image has been loaded into view.
+
+## ImageEditor inside stack layout
+
+### VerticalStackLayout
+
+The image editor is placed inside vertical stack layout, and users can specify a minimum height request, with the default value already predefined as 100.

```

Comment:
When the image editor is placed inside a vertical stack layout, users must define the required MinimumHeightRequest value. By default, this MinimumHeightRequest is set to 100.

Reply:
Updated